fudi-nfc/build.gradle

```diff
@@ -26,12 +26,12 @@ ext {
     allLicenses = ["Apache-2.0"]
 }
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    compileSdkVersion 31
```

Comment:
I can't build the project, Does it necessary to update kotlin & compileSdkVersion versions?